tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/UsernamePasswordIntegrationTests.cs

```diff
@@ -290,12 +292,6 @@ private async Task RunHappyPathTestAsync(LabResponse labResponse, string federat
             // 4) After successful log-in, remove the password line you added in with step 1, and run the integration test again.
         }
 
-        private void AssertCcsRoutingInformationIsSent(HttpSnifferClientFactory factory, LabResponse labResponse)
```

Comment:
The intention here is to make the code more readable by adding a method with a name that clearly identifies what is happening here. Otherwise it can be difficult for those not familiar with CCS to figureout exactly what we are doing.

Reply:
true. i didn't see the need for such a complicated method when i could just add a one-liner.